### PR TITLE
Delete user

### PR DESCRIPTION
**Pull request by danmassie**
Additional tests to follow, these changes will help with the edit user story so merging now.

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -49,3 +49,5 @@ editUser.countyStateHeader=County or State
 editUser.postcodeZipHeader=Postcode/Zipcode
 editUser.countryHeader=Country
 editUser.userGroupsHeader=User Groups
+
+deleteUser.success=User {0} has been deleted
```
Comment (susiecoleman) [inline conf/messages.en]: new line

**Comment by susiecoleman** [comment]
Looks good. But this could probably do with a second pair of eyes particularly on parts using flashes and RequestSigner.scala  before merging as I found those parts harder to understand

File: conf/messages.en (hunk 1 above)
Comment (danmassie) [inline conf/messages.en]: done

**Comment by danmassie** [comment]
@susiecoleman See https://www.playframework.com/documentation/2.4.x/ScalaSessionFlash for info on Flash scope.

**Comment by nlindblad** [comment]
LGTM, as long as the form with the delete call is not susceptible to CSRF :+1:

**Comment by markjamesbutler** [comment]
@nlindblad I started implementing CSRF on the forms in add-csrf-to-forms.  I was waiting for some forms to be added before I completed the implementation.  We can deal with the CSRF in the PR for add-csrf-to-forms when this PR is merged.

**Pull request merged**